cardano-node/cardano-node.cabal

```diff
@@ -81,6 +81,8 @@ library
                      , filepath
                      , formatting
                      , hostname
+                     , http-client
+                     , http-types
```

Comment:
where are these dependencies used?
maybe these could be moved to cardano-cli which actually does the http call?

Reply:
Unforyunately, these deps should be in main library, I cannot move it to executable `cardano-cli`.